validation/validationFunction/ww_validation_prototype.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+pip install cerberus
```

Comment:
pip commends should not be in the script. Instead, let's add a requirements.txt file at the root of the project listing the required packages and their versions.